src/clj/puppetlabs/http/client/async.clj

```diff
@@ -119,9 +121,38 @@
       (.setDecompressBody (clojure.core/get opts :decompress-body true))
       (.setHeaders (:headers opts))))
 
+(schema/defn get-mean :- schema/Num
+  [timer :- Timer]
+  (->> timer
+       .getSnapshot
+       .getMean
+       (.toMillis TimeUnit/NANOSECONDS)))
+
+(defn get-metric-data
+  [timer]
+  (let [count (.getCount timer)
+        mean (get-mean timer)
+        aggregate (* count mean)]
+    {:count count
+     :mean mean
+     :aggregate aggregate}))
```

Comment:
This is designed to match what is being returned by [trapperkeeper-comidi-metrics](https://github.com/puppetlabs/trapperkeeper-comidi-metrics/blob/6211f2621438cedc484ba0cd3c10d0d97a4a8106/src/puppetlabs/metrics/http.clj#L84-L98). It seems like it might be good to separate some of that out into a library (that doesn't have a trapperkeeper dependency) that could be used here.